prow/plugins/plugins.go

```diff
@@ -374,11 +374,12 @@ type Lgtm struct {
 	// compromised GitHub account) from merging arbitrary code. Use with caution.
 	//
 	// StickyForTrustedAuthors indicates if LGTM is sticky for PRs authored by trusted users, and
-	// eliminates the need to re-lgtm minor fixes/updates. This does not apply if the author is
-	// not trusted. Trusted users are by default collaborators and org members, but can also be
-	// restricted to only org members using Trigger.OnlyOrgMembers. i.e. the same set of users
-	// who can issue "/ok-to-test".
+	// eliminates the need to re-lgtm minor fixes/updates. TrustedAuthorTeam must also be configured
+	// to specify the team of trusted users. If StickyForTrustedAuthors is false, or if TrustedAuthorTeam
+	// is not set, lgtm is not sticky.
 	StickyForTrustedAuthors bool `json:"sticky_for_trusted_authors,omitempty"`
+	// Github team whose members are considered as trusted users for the sticky LGTM feature.
+	TrustedAuthorTeam string `json:"trusted_author_teams,omitempty"`
```

Comment:
Could we combine these two settings?

Reply:
It can be done, except that we have already turned on sticky_for_trusted_authors in our deployment.

However, we can fix up the config in our config when we rollout the new lgtm implementation.

Reply:
@fejta Done. PTAL again.